docs/docs-content/clusters/edge/local-ui/host-management/edit-user-data.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_label: "Edit User Data"
+title: "Edit User Data"
+description: "Instructions for editing user data in Local UI."
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["edge"]
+---
+
+The **user-data** file is a YAML file that contains installer configuration for the Palette agent. This file is usually
+prepared before the EdgeForge process and is written into the installer ISO in appliance mode deployments. In agent mode
+deployments, the user data is provided directly to the agent mode installation script. For more information about
+EdgeForge, refer to [EdgeForge Workflow](../../edgeforge-workflow/edgeforge-workflow.md). For more information about
+deployment modes, refer to [Deployment Modes](../../../../deployment-modes/deployment-modes.md).
+
+Local UI allows you to make edits to most fields in the **user-data** file after installation has completed before
+cluster creation. All fields exposed in the form in Local UI have corresponding parameters in the installer reference.
```

Comment:
```suggestion
In Local UI, you can interact with your user-data configuration and make updates to it. This is useful for situations in which you want to modify the user-data before forming a cluster with the Edge hosts. All fields exposed in the form in Local UI have corresponding parameters in the installer reference.
```
I wonder if maybe there is a way to make this sentence a bit easier to read 🤔  I took a stab at the sentence but feel free to change it